dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -62,30 +67,33 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
                 )
                 raise ValueError(msg)
         except Exception as err:
+            logger.error(f"Error while looking for required file {required_file}", 
+            err, 
+            data={"required_file": required_file, 
+            "required_file_patterns": required_file_patterns, 
+            "local_store": local_store})
```

Comment:
rather than passing in the store can we pass in a list of the files in the store? from memory there's a method to get them as a list, then just put it against a key of say "files_in_directory".

that way we've got the patterns we were looking for, the specific one that didn't match (from the error) and a list of the files considered.